recipes/restinstance/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "RESTinstance" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c241e44e51c7a793c7c5ca0476caac2b703cc78da8041ab05ded965ce49bc187
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
```

Comment:
This should be `noarch: python`

Reply:
I tried that, but than [the win-64 build fails](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=182256&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=26d9c833-f81c-502c-87bc-0cb0be44f838)

Reply:
That is OK.  Noarch: python only needs to build on linux.  It will run on any platform though

Reply:
```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
```